setup.py

```diff
@@ -18,5 +18,5 @@
     url='https://qiime2.org',
     long_description=long_description,
     packages=find_packages(),
-    install_requires=['click', 'flask', 'gevent', 'qiime2 == 2020.2.*']
+    install_requires=['click', 'flask', 'gevent', 'qiime2 == 2020.5.*']
```

Comment:
Please update the `version` key, too (should be `2020.5.0.dev0`) (up above on line 16 - I can't comment directly on the line).